common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1061,13 +1061,15 @@ def get_parent_location(self, locator, **kwargs):
             raise ItemNotFoundError(locator)
 
         course = self._lookup_course(locator.course_key)
-        parent_id = self._get_parent_from_structure(BlockKey.from_usage_key(locator), course.structure)
-        if parent_id is None:
+        parent_ids = self._get_parents_from_structure(BlockKey.from_usage_key(locator), course.structure)
+        if len(parent_ids) == 0:
             return None
+        # find alphabetically least
+        parent_ids.sort(key=lambda e: e.type + e.id)
```

Comment:
versus what? in a separate thread we said that get_parent and ilk should return the alphabetically least by usage key if there are more than one. Granted, there's no need to sort the list, just scan for the minimal, but having more than one should be an exception rather than the rule.